templates/dashboard/setting.html

```diff
@@ -26,7 +26,7 @@
     <div class="card">
       <div class="card-body">
         <div class="card-title mb-3">Current Plan</div>
-        {% if current_user.lifetime %}
+        {% if current_user.is_lifetime() %}
```

Comment:
Where is this function implemented? I cannot find it

Reply:
good one. fixed.